build.gradle.kts

```diff
@@ -59,11 +34,19 @@ subprojects {
 	apply(plugin = "com.coditory.manifest")
 	apply(plugin = "com.github.johnrengelman.shadow")
 
-	manifest {
+	project.extensions.getByType(ManifestPluginExtension::class.java).apply {
 		buildAttributes = false
 	}
 
-	lombok {
+	normalization {
+		runtimeClasspath {
+			metaInf {
+				ignoreManifest()
```

Comment:
Is this still needed given that line 38 disables the `buildAttributes` in the manifest?

Reply:
oh i see. the time stamp is no good for reproducibility.